Readme.md

```diff
@@ -1,29 +1,25 @@
-* [TestFlagsAttributeEnum.cs](./CS/EFCore/EnumCheckBoxEF/EnumCheckBoxEF.Module/BusinessObjects/TestFlagsAttributeEnum.cs)
-<!-- default file list end -->
-# How to represent an enumeration property via a drop-down box with check boxes
 
+# XAF - How to Display an Enumeration Property as a Drop-down Box with Check Boxes
 
-<p><strong>Scenario:</strong></p>
-<p>There is an enumeration type decorated with the FlagsAttribute, which means that an enumeration can be treated as a set of flags. There is also a property of this enumeration type inside the business class. This is helpful when several predefined enumeration values can be stored using this data property. This example shows how to create an editor to represent such a property via an editor with multiple check boxes:</p>
+This example demonstrates how to create a custom Property Editor that displays an enumeration property as a drop-down box with check boxes, if the enumeration type has `FlagsAttribute` that allows to treat this enumeration as a set of flags.
 
 ![image](https://user-images.githubusercontent.com/14300209/234846358-4435cbbe-130d-410b-9958-fb8450a11480.png)
 
-Steps To Implement:
-Since there is no <a href="https://documentation.devexpress.com/#Xaf/CustomDocument3552">standard  PropertyEditor for enumerations</a> that would allow you to store several values at once, it is common to <a href="http://documentation.devexpress.com/#Xaf/CustomDocument3097">implement custom Property Editors</a> for this task. A custom PropertyEditor will take data property value as its input and represent it in the UI using a custom visual control. There will be one PropertyEditor for WinForms (based on the <em>CheckedComboBoxEdit</em> control from the XtraEditors Suite) and one for Blazor (based on the [DxListBox](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxListBox-2) control from our Blazor Suite), because there is no platform-agnostic way to achieve such a look and feel.
+## Implementation Details
 
-<p><a href="http://documentation.devexpress.com/#Xaf/CustomDocument3097"><u>Implement Custom Property Editors</u></a>
+Since XAF does not have a built-in Property Editor for [enumeration properties](https://docs.devexpress.com/eXpressAppFramework/113552/business-model-design-orm/data-types-supported-by-built-in-editors/enumeration-properties) that can store several values at once,  you need to implement a custom Property Editor. It takes data property value as input and displays it in the UI with the help of a custom visual control. In an XAF Windows Forms application, the control is based on [`CheckedComboBoxEdit`](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.CheckedComboBoxEdit) editor from the XtraEditors Library. In an XAF ASP.NET Core Blazor application, it is based on the [DxListBox](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxListBox-2) control from the Blazor Library.
```

Comment:
We don't really use "Xtra..." names anymore. And the fact that the control comes from the "editors library" is not important either. A long time ago, a customer could buy an XtraEditors Library license separately. That's no longer possible. It's just one of our WinForms controls and no additional details are necessary. 

And since we won't mention the XtraEditors Library, no need for a parallel construct in the sentence about Blazor. 

```suggestion
XAF does not have a built-in Property Editor for [enumeration properties](https://docs.devexpress.com/eXpressAppFramework/113552/business-model-design-orm/data-types-supported-by-built-in-editors/enumeration-properties) that can store several values at once.  You need to implement a custom Property Editor that obtains the data property value and displays it in the UI with the help of a custom visual control. In an XAF Windows Forms application, the control is [`CheckedComboBoxEdit`](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.CheckedComboBoxEdit). In an XAF ASP.NET Core Blazor application, the control is [DxListBox](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxListBox-2).
```